#! /bin/sh /usr/share/dpatch/dpatch-run
## pdf_printing_support.dpatch by  <till.kamppeter@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: PDF as input format be added to the PPD files

@DPATCH@
diff -urNad hplip-2.8.7~/prnt/drv/hpijs.drv.in hplip-2.8.7/prnt/drv/hpijs.drv.in
--- hplip-2.8.7~/prnt/drv/hpijs.drv.in	2008-07-30 18:44:03.000000000 +0200
+++ hplip-2.8.7/prnt/drv/hpijs.drv.in	2008-08-13 12:38:17.000000000 +0200
@@ -84,7 +84,8 @@
 Throughput 1
 
 // CUPS filters...
-Filter "application/vnd.cups-postscript 0 @foomatic_filter@"
+Filter "application/vnd.cups-postscript 100 @foomatic_filter@"
+Filter "application/vnd.cups-pdf 0 @foomatic_filter@"
 
 {
   Manufacturer "Apollo"
